pkg/api/defaults-cloud-controller-manager.go

```diff
@@ -5,18 +5,27 @@ package api
 
 import (
 	"strconv"
+	"strings"
 
 	"github.com/Azure/aks-engine/pkg/api/common"
 )
 
 func (cs *ContainerService) setCloudControllerManagerConfig() {
 	o := cs.Properties.OrchestratorProfile
+	isAzureCNIDualStack := cs.Properties.IsAzureCNIDualStack()
+	clusterCidr := o.KubernetesConfig.ClusterSubnet
+	if isAzureCNIDualStack {
+		clusterSubnets := strings.Split(clusterCidr, ",")
```

Comment:
shouldn't we be splitting on `"."` (period)?, as the `ClusterSubnet` string property will be expressed as a normal CIDR string.

Reply:
Nope not splitting ip address here. Dual stack can have 2 subnets (v4 and v6) comma separated. Splitting v4 and v6 subnet here